rbc/src/main/java/com/library/rbc/model/User.java

```diff
@@ -0,0 +1,23 @@
+package com.library.rbc.model;
+
+import com.library.rbc.model.enums.Role;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "users")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class User {
+
+    @Id
+    private String id;
+    private String fullName;
```

Comment:
Suggestion: split this field into `firstName` and `lastName`.

Reply:
Same comment as for author full name.

Reply:
Resolved.